packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -1,8 +1,6 @@
 import { newE2EPage } from '@stencil/core/testing';
 import { axeCheck } from '../../../testing/test-helpers';
 
-const maxYear = new Date().getFullYear() + 100;
```

Comment:
Any particular reason this was removed? 

Edit: Lol ignore, I just checked and it isn't even being used anyway :shrug:

Reply:
🤔  This was in a test before. I wonder why/when it got removed? It used to be testing max year validation: https://github.com/department-of-veterans-affairs/component-library/pull/595#discussion_r1060971787

Reply:
I linked to a comment to `va-date` but it seems like there was a test for this compnent before too. Still not sure why it was removed before.

Reply:
I see now that it was updated for a translation string so the variable/value is now codified within i18next: https://github.com/department-of-veterans-affairs/component-library/pull/607/files 

Good clean up. Thanks!